pallets/anchors/src/lib.rs

```diff
@@ -571,7 +572,7 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
-	fn fee_key() -> <T as frame_system::Config>::Hash {
+	fn commit_fee_key() -> <T as frame_system::Config>::Hash {
 		<T as frame_system::Config>::Hashing::hash_of(&0)
```

Comment:
This `&0` should be changed to avoid collision between pallets. The initial purpose of this PR was to change this. 

@mikiquantum we were talking about creating an enum, which represents the numbers used in the hash, but thinking deeper into it I do not see a good place to define that enum without carrying a little knowledge of each pallet out of them. I think that maybe using a prefixed string for these cases could be simpler, the information is kept inside of each pallet. i.e. `hash_of("anchors::commit_fee")`, and there is no more collision between pallets. What do you think?

Reply:
@lemunozm The whole point is to have a central place where these keys are defined, so (1) developing the usage of a new fee is trivial and safe, and (2) these fee list (human readable - no hashes) are available to be read by a client (UI etc) so they can be included in tx_fee estimations. 
We could extend the fee storage to be indexed by an enum instead and that enum described the list of purposes (CommitAnchor, PreCommit, etc ...)

Reply:
Since the `pallet_fees` has to be imported by any other one that want to read a fee, then makes sense to add that enum list in that pallet or const variable.

Reply:
Ok, I see the point. I understood the `pallet_fees` as something isolated, without knowledge of how other pallets work (as an independent crate in the future). But yeah, definitely, the enum could be there, and that addition is simpler.